src/App.js

```diff
@@ -25,22 +26,22 @@ const alex = {
 
 const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
+    <Person
+      name={misha.name}
+      age={misha.age}
+      partner={misha.partnerName}
+    />
```

Comment:
```suggestion
    <Person person={ ...misha } />
```
Можна передавати цілий об'єкт і не розписувати кожне поле